heartbeat/docs/monitors/monitor-common-options.asciidoc

```diff
@@ -109,10 +107,10 @@ will not be executed by the scheduler.
 [[monitor-fields]]
 ==== `fields`
 
-Optional fields that you can specify to add additional information to the
-output. For example, you might add fields that you can use for filtering log
+These are optional fields that can be used to add additional information to the
```

Comment:
I don't like "that you can specify, " but changing to passive here is not the best fix. The sentence needs to be rewritten.